tests/conftest.py

```diff
@@ -4,15 +4,10 @@
 import pytest
 from ruamel.yaml import YAML
 
-from aiidalab.app import AiidaLabApp, _AiidaLabApp
-
 
 @pytest.fixture(scope="session")
 def static_path():
-    # TODO: Switch to importlib.resources once we drop support for Python 3.8
-    # import importlib.resources
-    # return importlib.resources.files() / "static"
```

Comment:
I've tried this but this approach does not work for Python < 3.12, because `importlib.resources.files()` requires a package name (not a module) and `tests/` is not a package (doesn't have `__init__.py` file).

https://docs.python.org/3/library/importlib.resources.html#importlib.resources.files

Reply:
(unrelated to this PR)